fe-games/src/components/Review.jsx

```diff
@@ -37,21 +60,29 @@ export function Review() {
         <p>Designed by {currentReview.designer}</p>
         <p>{currentReview.owner}</p>
         <p>{currentReview.review_body}</p>
-        <p>Votes: {currentReview.votes}</p>
+        {err ? <p>{err}</p> : null}
+        {voted ? <p>Removed vote!</p> : <p>Added vote!</p>}
+        <button onClick={() => upVote(currentReview.review_id)}>
+          Votes: {currentReview.votes}
```

Comment:
It might be better to have the votes displayed above the buttons. Or below? Including it in the button is a little confusing 😵‍💫